frontend/src/modules/activity/components/activity-form-drawer.vue

```diff
@@ -0,0 +1,453 @@
+<template>
+  <app-drawer
+    v-model="isVisible"
+    :size="480"
+    :title="isEdit ? 'Edit activity' : 'Add activity'"
+  >
+    <template #content>
+      <div class="-mt-4">
+        <app-form-item
+          class="mb-4"
+          label="Member"
+          :validation="$v.member"
+          :required="true"
+          :error-messages="{
+            required: 'This field is required'
+          }"
+        >
+          <el-select
+            v-model="form.member"
+            :filterable="true"
+            :remote="true"
+            placeholder="Select option"
+            :remote-method="searchMembers"
+            :loading="loadingMembers"
+            class="w-full"
+            @change="
+              selectMember($event) && $v.config.url.$touch()
+            "
+            @blur="$v.config.url.$touch"
+          >
+            <template
+              v-if="form.member && selectedMember"
+              #prefix
+            >
+              <app-avatar
+                :entity="{
+                  displayName: selectedMember.label,
+                  avatar: selectedMember.avatar
+                }"
+                size="xxs"
+              ></app-avatar>
+            </template>
+            <el-option
+              v-if="props.activity?.member"
+              :label="props.activity.member.displayName"
+              :value="props.activity.member.id"
+            >
+              <app-avatar
+                :entity="props.activity.member"
+                size="xxs"
+                class="mr-2"
+              ></app-avatar>
+              {{ props.activity.member.displayName }}
+            </el-option>
+            <el-option
+              v-for="item in membersList"
+              :key="item.id"
+              :label="item.label"
+              :value="item.id"
+            >
+              <app-avatar
+                :entity="{
+                  displayName: item.label,
+                  avatar: item.avatar
+                }"
+                size="xxs"
+                class="mr-2"
+              ></app-avatar>
+              {{ item.label }}
+            </el-option>
+          </el-select>
+        </app-form-item>
+
+        <app-form-item
+          class="mb-4"
+          label="When"
+          :validation="$v.datetime"
+          :required="true"
+          :error-messages="{
+            required: 'This field is required'
+          }"
+        >
+          <el-date-picker
```

Comment:
Could you add 
```
 class="custom-date-picker"
 popper-class="date-picker-popper"
```
to the date picker component? Currently it doesn't have the same style as the rest of the app